package.json

```diff
@@ -36,6 +36,7 @@
   },
   "dependencies": {
     "@babel/core": "7.5.5",
+    "@eyedea/flow-icons": "0.1.5",
```

Comment:
We should refrain of making `@smashing` dependent on anything Flow-related - Flow should be dependent on Smashing UI, not the other way. Also, Smashing is supposed to be open source (while Flow - not necessarily).
If icons are needed for storybook purpose, extract the code of 1-3 sample icons and put them in the story itself directly.

Reply:
done